HeterogeneousCore/AlpakaUtilities/interface/initialise.h

```diff
@@ -0,0 +1,27 @@
+#ifndef AlpakaCore_initialise_h
+#define AlpakaCore_initialise_h
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+namespace cms::alpakatools {
+
+  template <typename TPlatform>
+  void initialise();
```

Comment:
The implementation file of this function is missing.

Reply:
This is actually not used so I've removed it.